src/Parser.js

```diff
@@ -133,7 +135,19 @@ export default class Parser {
     parse(code: string): Program {
         let ast = this._parseAst(code);
         let tokens = this._processTokens(ast, code);
-        return buildElementTree(ast, tokens);
+        let program = buildElementTree(ast, tokens);
+        let programPlugins = {};
+        let plugins = this._options.plugins;
+        for (let plugin of plugins) {
+            let api = plugin.getProgramApi(program);
```

Comment:
Not every plugin will add Api to program (some of them will later affect parser behaviour).